src/sidebars/favourites/tearout.js

```diff
@@ -11,37 +11,53 @@
                         var dragElement = element[0],
                             tearElement = dragElement.parentNode.parentNode,
                             tileWidth = tearElement.clientWidth,
-                            tileHeight = tearElement.clientHeight;
+                            tileHeight = tearElement.clientHeight,
+                            store;
 
-                        function createConfig() {
+                        function createConfig(minWidth, minHeight, width, height, url, resizable, maximizable, showTaskbarIcon, saveWindowState) {
```

Comment:
Rather than pass the arguments in as one long function declaration, could you pass in a JSON object with those fields set, and add in `autoShow`/`frame` (or defaults for any undefined key)? That way, order isn't important and would make it easier to drop in/out keys.